script/debugger/debugtool.js

```diff
@@ -11,6 +11,15 @@ define('bigscreenplayer/debugger/debugtool',
      var view;
      var visible = false;
 
+     var LOG_LEVELS = {
+       ERROR: 0,
+       DEBUG: 1,
+       INFO: 2,
+       VERBOSE: 3
+     };
+
+     var logLevel = LOG_LEVELS.VERBOSE;
```

Comment:
Do we want this verbose by default? Won't this include all of the Dash.js debug when we are just using the debug tool?